ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/vlan-interfaces.j2

```diff
@@ -1,156 +1,166 @@
-{%            if vlan_interfaces[vlan_interface].pim.ipv4.sparse_mode is defined and vlan_interfaces[vlan_interface].pim.ipv4.sparse_mode == true %}
+{%       endfor %}
+{%     endif %}
+{%     if vlan_interfaces[vlan_interface].pim is arista.avd.defined %}
+{%        if vlan_interfaces[vlan_interface].pim.ipv4.sparse_mode is arista.avd.defined(true) %}
    pim ipv4 sparse-mode
-{%            endif %}
-{%            if vlan_interfaces[vlan_interface].pim.ipv4.local_interface is defined and vlan_interfaces[vlan_interface].pim.ipv4.local_interface is not none %}
+{%        endif %}
+{%        if vlan_interfaces[vlan_interface].pim.ipv4.local_interface is arista.avd.defined %}
    pim ipv4 local-interface {{ vlan_interfaces[vlan_interface].pim.ipv4.local_interface }}
-{%            endif %}
-{%         endif %}
-{%         if vlan_interfaces[vlan_interface].ipv6_virtual_router_address is defined and vlan_interfaces[vlan_interface].ipv6_virtual_router_address is not none %}
+{%        endif %}
+{%     endif %}
+{%     if vlan_interfaces[vlan_interface].ipv6_virtual_router_address is arista.avd.defined %}
    ipv6 virtual-router address {{ vlan_interfaces[vlan_interface].ipv6_virtual_router_address }}
-{%         endif %}
-{%         if vlan_interfaces[vlan_interface].isis_enable is defined %}
+{%     endif %}
+{%     if vlan_interfaces[vlan_interface].isis_enable is defined %}
    isis enable {{ vlan_interfaces[vlan_interface].isis_enable }}
-{%         endif %}
-{%         if vlan_interfaces[vlan_interface].isis_passive is defined and vlan_interfaces[vlan_interface].isis_passive == true %}
+{%     endif %}
+{%     if vlan_interfaces[vlan_interface].isis_passive is arista.avd.defined(true) %}
    isis passive
-{%         endif %}
-{%         if vlan_interfaces[vlan_interface].isis_metric is defined %}
+{%     endif %}
+{%     if vlan_interfaces[vlan_interface].isis_metric is arista.avd.defined %}
    isis metric {{ vlan_interfaces[vlan_interface].isis_metric }}
-{%         endif %}
-{%         if vlan_interfaces[vlan_interface].isis_network_point_to_point is defined and vlan_interfaces[vlan_interface].isis_network_point_to_point == true %}
+{%     endif %}
+{%     if vlan_interfaces[vlan_interface].isis_network_point_to_point is arista.avd.defined(true) %}
    isis network point-to-point
-{%         endif %}
-{%         if vlan_interfaces[vlan_interface].vrrp is defined and vlan_interfaces[vlan_interface].vrrp is not none %}
-{%           if vlan_interfaces[vlan_interface].vrrp.virtual_router is defined and vlan_interfaces[vlan_interface].vrrp.virtual_router is not none %}
-{%             if vlan_interfaces[vlan_interface].vrrp.priority is defined and vlan_interfaces[vlan_interface].vrrp.priority is not none and vlan_interfaces[vlan_interface].vrrp.priority > 1 %}
+{%     endif %}
+{%     if vlan_interfaces[vlan_interface].vrrp is arista.avd.defined %}
```

Comment:
with out latest findings, we don't need this check, since arista.avd.defined used below will check the full chain.

Reply:
Fixed with  07d5adb6